misc/influxdb_cq.py

```diff
@@ -16,21 +16,21 @@
 mysql_global_status_bytes_sent: {type: counter}
 mysql_global_status_connections: {type: counter}
 mysql_global_status_queries: {type: counter}
-node_cpu: {type: counter, group_by: 'mode, cpu'}
-node_disk_bytes_read: {type: counter, group_by: device}
-node_disk_bytes_written: {type: counter, group_by: device}
-node_disk_reads_completed: {type: counter, group_by: device}
-node_disk_writes_completed: {type: counter, group_by: device}
-node_filesystem_avail: {type: gauge, group_by: 'mountpoint, fstype, device'}
-node_filesystem_size: {type: gauge, group_by: 'mountpoint, fstype, device'}
+node_cpu_seconds_total: {type: counter, group_by: 'mode, cpu'}
+node_disk_read_bytes_total: {type: counter, group_by: device}
+node_disk_written_bytes_total: {type: counter, group_by: device}
+node_disk_reads_completed_total: {type: counter, group_by: device}
+node_disk_writes_completed_total: {type: counter, group_by: device}
+node_filesystem_avail_bytes: {type: gauge, group_by: 'mountpoint, fstype, device'}
+node_filesystem_size_bytes: {type: gauge, group_by: 'mountpoint, fstype, device'}
 node_load1: {type: gauge}
-node_memory_Buffers: {type: gauge}
-node_memory_Cached: {type: gauge}
-node_memory_MemAvailable: {type: gauge}
-node_memory_MemFree: {type: gauge}
-node_memory_MemTotal: {type: gauge}
-node_network_receive_bytes: {type: counter, group_by: device}
-node_network_transmit_bytes: {type: counter, group_by: device}
+node_memory_Buffers_bytes: {type: gauge}
+node_memory_Cached_bytes: {type: gauge}
+node_memory_MemAvailable_bytes: {type: gauge}
+node_memory_free_bytes: {type: gauge}
+node_memory_total_bytes: {type: gauge}
```

Comment:
Please replace these metrics to
node_memory_MemFree_bytes
node_memory_MemTotal_bytes

![Screenshot_20200319_105131](https://user-images.githubusercontent.com/25184101/77048951-e9360600-69cf-11ea-9dae-cc276a594617.png)

Reply:
done !